aula-pratica-03/ex9.c

```diff
@@ -0,0 +1,51 @@
+#include<stdio.h>
+int find_menor(int *voltas, int n);
+int menor_volta(int *voltas,int menor, int n);
+int media(int *voltas, int n);
+
+int find_menor(int *voltas, int n) {
+    int menor = voltas[0];
+    int menor_volta = 0;
+    for(int i = 0; i < n; i++) {
```

Comment:
```suggestion
    for(int i = 1; i < n; i++) {
```
you already is processing the first item in the first line of this function